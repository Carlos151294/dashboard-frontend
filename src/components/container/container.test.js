import React from 'react';
import { shallow } from 'enzyme';
import Container from './container.component';

it('Excepect to render Container component with no classname property', () => {
    expect(shallow(<Container />)).toMatchSnapshot();
});

it('Excepect to render Container component with classname property', () => {
    expect(shallow(<Container className="my-class"/>)).toMatchSnapshot();
});