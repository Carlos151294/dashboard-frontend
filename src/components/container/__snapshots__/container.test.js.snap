// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Excepect to render Container component 1`] = `ShallowWrapper {}`;

exports[`Excepect to render Container component with classname property 1`] = `ShallowWrapper {}`;

exports[`Excepect to render Container component with no classname property 1`] = `ShallowWrapper {}`;
